src/interfaces/IMorpho.sol

```diff
@@ -67,12 +67,18 @@ interface IMorpho {
     function feeRecipient() external view returns (address);
 
     /// @notice The state of the position of `user` on the market corresponding to `id`.
+    /// @dev Warning: `supplyShares` does not contain the additional shares accrued by `feeRecipient` since the last
+    /// interest accrual.
```

Comment:
```suggestion
    /// @dev Warning: For `feeRecipient`, `supplyShares` does not contain the additional shares accrued since the last
    /// interest accrual.
```